assets/gaming/kittyVerse.json

```diff
@@ -0,0 +1,21 @@
+{ 
+  "metadata": { 
+    "label": "kitty_verse", 
+    "name": "kittyVerse", 

```

Comment:
Please use the name `KittyVerse` and organization `kitty_verse`.

Reply:
also please check the CI errors 

https://github.com/ton-studio/ton-labels/actions/runs/14516184518/job/40727733446?pr=248
<img width="909" alt="image" src="https://github.com/user-attachments/assets/fd44c8e3-182a-44e3-8722-488b6184b8b0" />